lib/honeybadger/config/ruby.rb

```diff
@@ -95,6 +105,28 @@ def exception_fingerprint
         hash[:exception_fingerprint] = Proc.new if block_given?
         get(:exception_fingerprint)
       end
+
+      private
+
+      def validate_before_action(action)
```

Comment:
We're doing two separate checks, so an `if/elsif/else` seemed optimal even though I tend to not like them. What do you think?

Reply:
The current implementation looks good :+1: